rabbitmq/src/main/resources/definitions.json

```diff
@@ -37,7 +38,8 @@
       "pattern": ".*healthcheck.*",
```

Comment:
(this is not a change introduced by this PR and shouldn't hold up the merge, but might be relevant for testing the PR)

question: how many queue(s) do we anticipate this matching? I think the answer is one, but I wanted to double-check.

Browsing rabbitmq on sandbox, i see we have a queue named `healthcheck`; and in the default branch i see  in [constants.rb](vro/blob/23c883a536a36545acead4267f38645723e51d66/svc-bgs-api/src/config/constants.rb) a queue named `svc_bgs_api.healthcheck` (and i think it's just not deployed). regarding the pattern match, is the idea that it'll match the latter?  if so, should the regex be `*healthcheck`?

Reply:
So that `pattern` is a regex pattern, and `".*healthcheck.*"` will actually match both of those. I'll be documenting some of this in the wiki when I get a chance today, but essentially, this will match any services "healthcheck" queues, and yes, currently only bgs has one.